OWL2/AS.hs

```diff
@@ -557,5 +565,122 @@ data ClassExpression =
 data Annotation = Annotation [Annotation] AnnotationProperty AnnotationValue
     deriving (Show, Eq, Ord, Typeable, Data)
 
-data AnnotationValue = AnnValue IRI | AnnValLit Literal
+type OntologyAnnotations = [Annotation]
+
+
+data AnnotationValue =
+    AnnAnInd AnonymousIndividual
+  | AnnValue IRI
+  | AnnValLit Literal
     deriving (Show, Eq, Ord, Typeable, Data)
+
+data AnnotationAxioms = 
+    AnnotationAssertion AxiomAnnotations AnnotationProperty AnnotationSubject AnnotationValue 
+  | SubAnnotationPropertyOf AxiomAnnotations SubAnnotationProperty SuperAnnotationProperty 
+  | AnnotationPropertyDomain AxiomAnnotations AnnotationProperty IRI
+  | AnnotationPropertyRange AxiomAnnotations AnnotationProperty IRI
+
+-- Annotation Assertion
+data AnnotationSubject = AnnSubIri IRI | AnnSubAnInd AnonymousIndividual
+
+-- Annotation Subproperties
+type SubAnnotationProperty = AnnotationProperty
+type SuperAnnotationProperty  = AnnotationProperty
+
+
+-- * AXIOMS
+
+data Axiom =
+  Declaration
+  | ClassAxiom_ ClassAxiom
```

Comment:
I didn't manage to understand why you need this underscore here. If you have a good reason, please add a comment. If it can be removed, please remove it.